package.json

```diff
@@ -165,6 +165,7 @@
     "js-yaml": "^3.14.0",
     "json-refs": "^3.0.15",
     "mockdate": "3.0.2",
+    "nodemon": "^2.0.4",
```

Comment:
We already have [`sane`](https://www.npmjs.com/package/sane) required, can we use that instead?